src/llhttp/http.ts

```diff
@@ -313,8 +321,21 @@ export class HTTP {
     };
 
     // Response
+    const endResponseProtocol = () => {
+      return this.span.protocol.end(
+        this.invokePausable('on_protocol_complete', ERROR.CB_PROTOCOL_COMPLETE, n('res_after_protocol'),
+        ));
+    }
+
     n('start_res')
-      .match('HTTP/', span.version.start(n('res_http_major')))
+      .otherwise(this.span.protocol.start(n('res_after_start')));
+
+    n('res_after_start')
+      .match([ 'HTTP', 'RTSP', 'ICE' ], endResponseProtocol())
+      .otherwise(this.span.protocol.end(p.error(ERROR.INVALID_CONSTANT, 'Expected HTTP/')));
+
+    n('res_after_protocol')
+      .match('/', span.version.start(n('res_http_major')))
       .otherwise(p.error(ERROR.INVALID_CONSTANT, 'Expected HTTP/'));
```

Comment:
Actually expects now `HTTP, RTSP or ICE` (also without the `/`)

Reply:
Done